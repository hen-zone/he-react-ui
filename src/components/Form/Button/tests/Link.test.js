// @flow
import createTestContext from 'react-cosmos-test/enzyme';
import fixture from '../fixtures/linkStyle.fixture';

const { mount, getWrapper } = createTestContext({ fixture });

beforeEach(mount);

test('<Button /> rendered correctly with fixture', () => {
  const wrapper = getWrapper();
  expect(wrapper.find('button')).toHaveLength(1);

  expect(wrapper).toMatchSnapshot();
});
