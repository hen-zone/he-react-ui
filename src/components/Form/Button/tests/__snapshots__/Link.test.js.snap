// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Button /> rendered correctly with fixture 1`] = `
<Button
  color="teal"
  disabled={false}
  icon=""
  keyline={false}
  link={true}
  onClick={[Function]}
  squared={false}
  submit={false}
  withRouter={false}
>
  <div
    className="buttonContainer"
  >
    <InnerButton
      className="button teal link"
      onClick={[Function]}
      submit={false}
      withRouter={false}
    >
      <button
        className="button teal link"
        onClick={[Function]}
        type="button"
      >
        <div
          className="content"
        >
          Button
        </div>
      </button>
    </InnerButton>
  </div>
</Button>
`;
